addons/base_import_module/models/ir_module.py

```diff
@@ -295,6 +296,7 @@ def _import_zipfile(self, module_file, force=False, with_demo=False):
                         if os.path.splitext(filename)[1].lower() not in ('.xml', '.csv', '.sql'):
                             continue
                         module_data_files[mod_name].append('%s/%s' % (mod_name, filename))
+                    dependencies[mod_name].extend(terp.get('depends', []))
```

Comment:
Is it really legal to have multiple manifests for one module? and do we really need to merge those depends or can simply use the last one?
Usually I expect `dependencies = {}` and  `dependencies[mod_name] = terp.get('depends') or ['base']`
Note: `['base']` is not really necessary in your code. It is semantically consistent with how we fallback for normal modules' manifests

Reply:
No you're right. And since https://github.com/odoo/odoo/commit/298ecdb77422ae726c9650d1c242d97621ed799b the only possible value name is `__manifest__.py`. I'll update the code accordingly.